org.zotero.Zotero.json

```diff
@@ -0,0 +1,41 @@
+{
+    "id": "org.zotero.Zotero",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "zotero",
+    "rename-desktop-file": "zotero.desktop",
+    "rename-icon": "zotero",
+    "finish-args": [
+        "--socket=x11",
+        "--share=ipc",
+        "--share=network",
+        "--filesystem=home"
+    ],
+    "modules": [
+        {
+            "name": "zotero",
+            "buildsystem": "simple",
+            "sources": [{
+                "type": "archive",
+                "url": "https://download.zotero.org/client/release/5.0.34/Zotero-5.0.34_linux-x86_64.tar.bz2",
+                "sha256": "df0c5aa38b7e2ec2a41006f7cb8616fce3956f452c19125dc89c66b11c51e392"
+            },
+            {
+                "type": "file",
+                "path": "org.zotero.Zotero.appdata.xml"
+            }],
+            "build-commands": [
+                "mkdir -p /app/bin /app/share /app/share/icons/hicolor/{16x16,32x32,48x48,256x256}/apps /app/share/appdata",
```

Comment:
Are there 64x64 or 128x128px icons? If you don't have these in the appdata then the app won't look ideal in software stores or the flathub website.

Reply:
I think they ship a scalable icon, I'll check that later.